### PR TITLE
Dataset for the dashboard

### PR DESCRIPTION
**Pull request by jananibabu3**
## What are the changes?

- Added a new dataset for heart rate and SP02 variables, which will be used for developing the health monitoring dashboard.

## Who worked on the changes?

-  Aryan, Janani and Karthik worked on sourcing, preparing, and uploading the dataset.

## New Components/Features

- Uploaded an extensive dataset (1000 entries) containing health-related variables such as heart rate, SP02, and others to be utilized in building and refining the health monitoring dashboard.

## Enhanced Components/Features

- NA

## Other Changes/Fixes

- NA

## Additional Notes

-  The dataset provides a broader scope for testing and validating the dashboard's functionalities across various health metrics

## Checklist

- [x] Existing, relevant documentation has been updated to reflect my changes

### CODE REVIEW

**Comment by Manan2711** [comment]
Hi @jananibabu3, can you please make the following changes:

1. Please add the data file into a subfolder rather than adding it directly into the "Health Monitoring Dashboard" folder.
2. In the data folder please add a markdown file that provides the following details:
\- The units in which the data is recorded, for example celsius for temperature columns and so on.
\- Just a short description of the dataset – where it was sourced from (add a link to the source), what each columns are about, etc.
4. In main README.md file (the one in "Health Monitoring Dashboard" folder) – please include a short description on what the feature is about, folder structure, etc rather than short updates. Please refer to [this](https://deakin365.sharepoint.com/:t:/r/sites/RedbackOperations9/Shared%20Documents/Project%202%20-%20Elderly%20Wearable%20Technology/[README_TEMPLATE.MD](https://deakin365.sharepoint.com/:t:/r/sites/RedbackOperations9/Shared%20Documents/Project%202%20-%20Elderly%20Wearable%20Technology/README_TEMPLATE.MD?csf=1&web=1&e=98jIYI)?csf=1&web=1&e=98jIYI) markdown file to get ideas on what to include in the readme file.

**Comment by ben-AI-cybersec** [comment]
No code, just data - doesn't require secdevops review.

File: Data Science/Health Monitoring Dashboard/HealthData.xlsx
Comment (Manan2711) [inline Data Science/Health Monitoring Dashboard/HealthData.xlsx:1]: I suppose this file was accidently added to the main folder as well. Can you please delete this file from here?

File: Data Science/Health Monitoring Dashboard/Dataset/sensor data
Comment (Manan2711) [inline Data Science/Health Monitoring Dashboard/Dataset/sensor data:1]: Was this file added accidently? It's an empty file, can you please check this one too.

**Review by Manan2711 — CHANGES_REQUESTED** [review]

File: Data Science/Health Monitoring Dashboard/Dataset/sensor data
Comment (jananibabu3) [reply Data Science/Health Monitoring Dashboard/Dataset/sensor data:1]: Updated now!

**Pull request merged**